tests/unit/models/BucketInfo.spec.js

```diff
@@ -694,6 +699,12 @@ Object.keys(acl).forEach(
                 assert.deepStrictEqual(
                     dummyBucket.getCapabilities(), testCapabilities);
             });
+            it('setQuota should set bucket quota', () => {
```

Comment:
We can also test when nothing is specified: it should still work with old (already existing) buckets, hence we must ensure we handle them properly

Reply:
here's a test with a test quota and no value provided => defaults to 0 https://github.com/scality/Arsenal/pull/2234/commits/a5541f28bc6eb5fe46e0ac5e658a2be100057e15#diff-31604c11f0c75e4caa3bd813de17aa4ff66eee7a586dd1a583acd1e1feaf48e6R708-R712